modules/affile/logproto-file-writer.c

```diff
@@ -32,6 +33,8 @@
 typedef struct _LogProtoFileWriter
 {
   LogProtoClient super;
+  SignalSlotConnector *signal_slot_connector;
+  AFFileDestDriver *signal_slot_connector_driver;
```

Comment:
We should avoid storing or referencing `AFFileDestDriver` in this class, if possible.

Reply:
:+1: